db/people.yml

```diff
@@ -1107,22 +1107,17 @@ sharouna-mayer:
 smayers:
   active: true
   aka:
-    - Sam R. Mayers
+    - Sam Mayers
   avatar: https://github.com/samrmayers.png
-  bio: bio
+  bio: Sam is a junior at Columbia University studying Applied Math.
   education:
-    - begin_year: 2011
+    - begin_year: 2017
       begin_month: 9
-      degree: PhD in Physics
+      degree: Current undergrad
       institution: Columbia University
-      end_year: 2016
-      end_month: 4
+      end_year: 2021
+      end_month: 5
       group: bg
-  employment:
```

Comment:
The way we have been doing it we put your stint doing undergrad research in the group as an employment.  Make position sthg like ``Undergraduate Researcher`` and please add ``group: bg`` in that employment item and the builder programs will find it and recognize that you were in the group during that time